src/Schema/PostgreSQLSchemaManager.php

```diff
@@ -104,26 +66,24 @@ final protected function getExistingSchemaSearchPaths(): array
      */
     protected function getCurrentSchema(): ?string
     {
-        $schemas = $this->getExistingSchemaSearchPaths();
+        if ($this->currentSchema === null) {
+            $this->currentSchema = $this->determineCurrentSchema();
+        }
 
-        return array_shift($schemas);
+        return $this->currentSchema;
```

Comment:
```suggestion
        return $this->currentSchema ??= $this->determineCurrentSchema();
```

We don't need the `if` block on PHP 7.4. 🤓

Reply:
Nice catch! Also, see the last commit.